spec/features/chooser/chooser_to_filter_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+describe 'Chooser To Filter', js: true do
+  before do
+    collections_response = Cmr::Response.new(Faraday::Response.new(status: 200, body: JSON.parse(File.read('spec/fixtures/cmr_search.json'))))
```

Comment:
I made a small helper, `cmr_success_response`, to make this a bit easier, but I don't think it's used in all places (yet). If you want to use it, this could be changed to `collections_response = cmr_success_response(File.read('spec/fixtures/cmr_search.json'))`